tools/protovalidate-conformance/internal/cases/cases_is_ip.go

```diff
@@ -284,6 +284,14 @@ func isIPSuite() suites.Suite {
 			Message:  &cases.IsIp{Val: "::"},
 			Expected: results.Success(true),
 		},
+		"version/omitted/valid/ipv6/j": {
+			Message:  &cases.IsIp{Val: "0:00::000:0000"},
+			Expected: results.Success(true),
```

Comment:
Just for context (I agree with adding this test): RFC 5952 gives a recommendation for a canonical text representation in [section 4](https://datatracker.ietf.org/doc/html/rfc5952#section-4).

Reply:
SGTM. Makes me wonder if we should also have a "canonical" IPv6 validator.